e2e/wdio/wdio.local.conf.ts

```diff
@@ -66,3 +61,12 @@ if (os.platform() === 'darwin') {
         browserName: 'safari'
     })
 }
+
+if (os.platform() === 'win32') {
```

Comment:
We have Edge support in all OS'es so we should test the ability to set this up in all as well.

Reply:
Oh right since it is Chromium based, totally forgot that. I'll take a look at it tomorrow, thanks :)